int16 strain0
uint8 level0
int16 strain1
uint8 level1
int16 strain2
uint8 level2
int16 strain3
uint8 level3
